griptape/artifacts/audio_artifact.py

```diff
@@ -15,6 +19,12 @@ class AudioArtifact(BlobArtifact):
 
     format: str = field(kw_only=True, metadata={"serializable": True})
 
+    def __attrs_post_init__(self) -> None:
+        # Generating the name string requires attributes set by child classes.
+        # This waits until all attributes are available before generating a name.
+        if self.name == self.id:
+            self.name = self.make_name()
```

Comment:
Out of scope for this PR since we just need to fix it, but could an alternative be to make a `name` a `@property`? Might be a little more intuitive / readable